vector/src/main/java/im/vector/app/core/services/VectorService.kt

```diff
@@ -33,10 +31,6 @@ abstract class VectorService : Service() {
      */
     private var mIsSelfDestroyed = false
 
-    override fun attachBaseContext(base: Context) {
```

Comment:
Why is this removed?

Reply:
It is crashing the application. @bmarty suggested to remove it.

Reply:
@bmarty any thought?

Reply:
This is OK to remove it, I think this is not really necessary for Services. It was added recently (after the previous release) to have a localized Context. It's probably useless for Services.